packages/hyperproxy-node/src/index.js

```diff
@@ -28,11 +28,20 @@ class HyperproxyNode {
 
     async _connectToHub(key) {
         const client = new HyperproxyHubClient(w3cwebsocket, Wrtc)
-        client.hub.subscribe(key).on('data', async ({sender, message}) => {
+        client.hub.subscribe(key).on('data', async ({sender, message, type}) => {
             try {
+                if (type === 'response') {
+                    return;
+                }
                 const file = await this.readFile(message);
-                sender.emit(file);
+                client.hub.broadcast(key, {
+                    sender: client.swarm.me,
+                    type: 'response',
```

Comment:
Should refactor these into some kind of enum, maybe put it in the config module?